cmd/atlas/atlas.go

```diff
@@ -99,21 +94,49 @@ func createConfigFromMongoCLIConfig() {
 	_, err = io.Copy(out, in)
 	if err != nil {
 		log.Printf("There was an error generating %s: %v", atlasConfigPath, err)
+		return
 	}
 
 	_, _ = fmt.Fprintf(os.Stderr, `AtlasCLI has copied your MongoCLI configuration to: %s
 
 `, atlasConfigPath)
 }
 
-func mongoCLIConfigPath() string {
+func mongoCLIConfigPaths() (configPath, oldConfigPath string) {
```

Comment:
is this method necessary? you're probably only gonna use one, can we just call them separately instead of a method to get both at the same time?

Reply:
sure!